### ISSUE TITLE
Finding a reasonable margin between players holding cards and using alts

### ISSUE THREAD

**Issue by Muteki9Soul** [Marenthyu/NepBot#58]
This has been a long standing grey area, and because many players already practice things be it holding cards for each other, bounties, pacts, etc, its has remained difficult to draw a line.

**Comment by Marenthyu** [comment]
I dont think you meant to close this. If you want to add more comments, just do so, no need to add a new issue. (Also closing != deleting)

**Comment by xorhash** [comment]
Shouldn't this be closed, though? This is a GitHub repo. I'm under the impression the GitHub issues for this repository are for features/bugs of the implementation, not for policy decisions.

**Comment by WillLi05** [comment]
The biggest problem is actually people who are 'just barely active' who hold cards for people when you look at this from an outside perspective. So rather than worrying about how active they are, probably best to come up with some way of telling what they're doing more. "Playing for themselves" or "Playing for others"

**Comment by wherewedroppinboys** [comment]
I think, firstly, there should be a distinction between holding cards and bountying cards for other players, and essentially owning a second hand. Not only are you trusting someone by letting them hold a card for you, there's a massive difference between one hand space and having an entirely separate hand to 
"bank with" at will. As far as pacts are concerned, this requires trust as well and is a totally normal part of any game.

The problem with this is that in order to take action on behavior like this, the solutions would all be invasive in some shape or form. That being said, there are a few ways, hypothetically, to either force "alt hand players" to participate actively in TCG, or to hinder the ability to have a player essentially function as a second hand (and as a note, these are just ideas, even though I'm mentioning them here it doesn't mean I personally support them):

1) Implement a new system that allows players to "bank" cards out of their hand for a flat fee that's above the cost of DE value in order to prevent abuse. This could be 1/6th to 1/3rd of their current hand size. Also charge them a point fee to withdraw cards, or place a time restriction on how long a card can be "banked" for if necessary. This would eliminate the use of alt accounts entirely.

2) Implement a Discord verification system on the bot in order to enable a player to trade cards above a certain rarity; ie, require a player to join any TCG-related Discord server and have them input a verification command which could be checked on waifus.de - this would essentially support muteki's original idea in the OP here, but would also limit players who aren't interested in joining Discord from using a significant aspect of TCG.

3) Use admin discretion to identify players who function as "alt hand players" and disallow the practice completely; as well, if you really wanted, add a public log for every mythical card traded between players so if a user spots someone doing this they can make the staff aware. This would be problematic as it would expose MUH SHADY DEALS and the actions of genuine players, but I figured I'd throw the idea in regardless. 

Just a few ideas. I think overall the best thing to do would be to have the staff monitor it actively once they decide on an official stance on the matter.
